src/components/body/body.js

```diff
@@ -127,16 +128,6 @@ var Body = {
   update: function (prevData) {
     var data = this.data;
 
-    if (data.type !== prevData.type) {
-      if (data.type === 'dynamic' && data.mass === 0) {
-        this.el.setAttribute(this.name, {mass: 5});
-      }
-      if (data.type === 'static' && data.mass !== 0) {
-        this.el.setAttribute(this.name, {mass: 0});
-      }
-      return;
-    }
```

Comment:
i expect we need to do something if `data.type` changes, no? Not sure if `body.type` can actually be changed after the fact, if not we should log a warning...

Reply:
I don't think you can actually change the type. The only way to effectively switch between dynamic and static is by starting with dynamic and changing the mass from non-zero to zero. So you are probably right, a warning should be logged.